tests/vm_test.go

```diff
@@ -785,7 +787,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			})
 
 			Context("Using RunStrategyHalted", func() {
-				It("should start a stopped VM", func() {
+				It("[test_id:2037] should start a stopped VM", func() {
```

Comment:
I am missing the action of  start VM and check vmi , make sure RunStrategy=always 
Ensure vm.status.stateChangeRequests is not present in vm's manifest

Reply:
Confirmed test is correct in offline conversation with @ILpinto